xtas/core.py

```diff
@@ -68,8 +68,8 @@ def configure(config, import_error="raise", unknown_key="raise"):
             else:
                 logger.warn(msg)
 
-    app.config_from_object(config.get('CELERY', 'xtas._defaultconfig.CELERY'))
```

Comment:
That makes sense, those are a bit odd. Another thing I noticed is that _defaultconfig.py uses dict() instead of just { }, which seems to have come from the Celery docs as well. I'll leave it up to yu whether you want things more Pythonic or more Celeric...